scripts/deploy.ts

```diff
@@ -1,30 +1,28 @@
 import hre from "hardhat";
 
 async function main() {
-    const [deployer, gameOwner] = await hre.ethers.getSigners();
+    const [deployer] = await hre.ethers.getSigners();
 
     // Governance Token deployment
     console.log("Deploying Governance contract with the account:", deployer.address);
     console.log("Account balance:", (await deployer.getBalance()).toString());
 
     const governanceTokenFactory = await hre.ethers.getContractFactory("GovernanceToken");
 
+    const addressList = [process.env.EXCHANGES_WALLET_ADDRESS, process.env.PLAYANDEARN_WALLET_ADDRESS,
+        process.env.SOCIAL_WALLET_ADDRESS, process.env.TEAM_WALLET_ADDRESS, process.env.TREASURY_WALLET_ADDRESS, process.env.ADVISORS_WALLET_ADDRESS];
+
     const governanceToken = await governanceTokenFactory.deploy(
-        8000000000000000, "MEE Governance Token", 18, "MEE", process.env.GAME_OWNER_ADDRESS, {gasLimit: 6e6});
+        8000000000000000, "MEE Governance Token", 18, "MEE", process.env.GAME_OWNER_ADDRESS,
```

Comment:
can we make the 8000000000000000 something in the contract itself, and there then with 8_000_000_000_000_000.

Also it's a weird number (I guess it should be 3 billion?)

I suggest to remove the parameter and have it all configured in the contract.

Reply:
Fixed